### DISCUSSION TITLE
SMTP Error: Could not connect to SMTP host.

### DISCUSSION THREAD

**Discussion by mucktobrak** [postalserver/postal#2422] [Help with using Postal]
When I connect to auto, I get this error, but when I get security on auto on gmass.co, it sends to auto. How do I integrate it into auto?
![Ekranddd](https://user-images.githubusercontent.com/131215796/233873089-d8652e4d-953a-41c0-b78a-59de9d30eb00.JPG)
![Ekran Alınsddddddtısı](https://user-images.githubusercontent.com/131215796/233873090-f6a8967f-6abd-4f79-ba0c-e81538ff1ca3.JPG)

**Comment by mucktobrak** [comment]
It works like that in the photo I mentioned above, how do I integrate it into my auto?

**Comment by willpower232** [comment]
Can you telnet to your Postal server from another computer with `telnet yourpostaldomain.com 25`?

**Comment by mucktobrak** [comment]
how can i do it
